at_client/lib/src/transformer/request_transformer/notify_request_transformer.dart

```diff
@@ -0,0 +1,90 @@
+import 'dart:async';
+
+import 'package:at_client/src/preference/at_client_preference.dart';
+import 'package:at_client/src/service/notification_service.dart';
+import 'package:at_client/src/transformer/at_transformer.dart';
+import 'package:at_client/src/util/at_client_util.dart';
+import 'package:at_commons/at_builders.dart';
+import 'package:at_commons/at_commons.dart';
+import 'package:at_client/src/encryption_service/encryption.dart';
+
+/// Class is responsible for taking the [NotificationParams] and converting into [NotifyVerbBuilder]
+class NotificationRequestTransformer
```

Comment:
I'd like to see some unit tests on this new class please.

Reply:
I have added unit tests in notification_service_test.dart file. May i know if the comments mean to add few more test cases?